packages/llamaindex/src/readers/LlamaParseReader.ts

```diff
@@ -207,6 +213,15 @@ export class LlamaParseReader extends FileReader {
       vendor_multimodal_api_key: this.vendorMultimodalApiKey,
     };
 
+    // Filter out invalid specific parameters
+    const filteredParams = this.filterSpecificParams(LlamaParseBodyParams, [
```

Comment:
where is this used?

Reply:
Oh yeah.. I did forget to actually use it.